modules/ossm-remove-cleanup.adoc

```diff
@@ -87,7 +87,7 @@ $ oc delete cm -n openshift-operators maistra-operator-cabundle
 +
 [source,terminal]
 ----
-$ oc delete cm -n openshift-operators istio-cni-config
+$ oc delete cm -n openshift-operators istio-cni-config-v2-3
```

Comment:
```suggestion
$ oc delete cm -n openshift-operators istio-cni-config istio-cni-config-v2-3
```
In addition to deleting the `istio-cni-config-v2-3` ConfigMap, the `istio-cni-config` must also be deleted if the user had installed an SMCP v2.2 or earlier. 

When 2.4 is released, we'll also need to add `istio-cni-config-v2-4` here, but we may be able to use a label selector instead of all three names, but I don't know the labels off the top of my head.